### PR TITLE
Allows users to set display size via parameters

### PR DESCRIPTION
**Pull request by flatsiedatsie**
This feature allows a user to set a prefered display size via a command line parameter. E.g.

`rpiplay -w 1280 -h 720`

It works, kinda...

For example, I was trying to get the display to default to 1024x600, but it seems my macbook will accept well known resolutions, but not ones it doesn't consider 'normal'. So:

`rpiplay -w 1280 -h 720` -> select "default for display" -> works
`rpiplay -w 1920 -h 1080` -> select "default for display" -> works
`rpiplay -w 1024 -h 600` -> select "default for display" -> 1280x720

My first thought was that it selects 1280x720 because is the nearest resolution with the same aspect ratio to 1024x600. However, even aspect ratios the macbook shows as 'scaled' options don't immediately lead to them being set.

For example, if you set `rpiplay -w 1440 -h 900`, which is a supported scaled resolution, and then select "default for display", it changes it to 1280x720 again. The same thing happens with 800x600 and 1024x768.

I'm not sure what's going on here. But I thought I'd share the code anyway.

### CODE REVIEW

**Comment by davidschlegel** [comment]
This currently does not work for me on gstreamer because ilclient is missing. Do you know whether this will be merged with the master soon?

File: rpiplay.cpp [hunk 1]
```diff
@@ -145,6 +154,10 @@ int main(int argc, char *argv[]) {
             low_latency = !low_latency;
         } else if (arg == "-r") {
             rotation = atoi(argv[++i]);
+        } else if (arg == "-w") {
+            display_width = atoi(argv[++i]);
+        } else if (arg == "-h") {
+            display_height = atoi(argv[++i]);
```
Comment (markuspi) [inline rpiplay.cpp:160]: `-h` is conflicting with the help command

Comment (FD-) [reply rpiplay.cpp:160]: Right. I'd prefer a single flag for both width and height, something like -s 1280x720.

**Comment by seungjunProgramming** [comment]
Does this make 4K res. available???

**Pull request closed without merging**